apps/kaizen/models.py

```
@@ -1,3 +1,56 @@
+import uuid
 from django.db import models
+from apps.accounts.models import User
+from common.models import BaseModel
 
-# Create your models here.
+
+# Task model
+class Task(BaseModel):
+    title = models.CharField(max_length=255)
+    description = models.TextField(blank=True, null=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `TextField`

(DJ001)